data/templates/fail2ban/jail.conf

```diff
@@ -581,4 +581,5 @@ enabled  = true
 port     = http,https
 protocol = tcp
 filter   = yunohost
-logpath  = /var/log/nginx/*.log
+logpath  = /var/log/nginx*/*error.log
```

Comment:
I checked on 2 distinct yunohost, I have one `nginx` directory but no need `nginx*`. Are you sure it's needed ?